README.md

```diff
@@ -165,22 +33,18 @@ my_package/my_module.py
 tests/test_functionality.py
 ```
 
-## String quotes
-
-Use double quotes for human-readable strings and single quotes for strings used as keys or tokens.
-
-```python
-logger.info("this is a human-readable message")
-raise RuntimeError("something has gone wrong")
-
-my_dict['my_key'] = my_value
-raise ApiError(400, {'code': 'invalid_value'})
-```
-
 ## Section dividers
```

Comment:
i wouldn't be averse to getting rid of this peculiarity. it's a weird thing i picked up from someone at my last job, and i don't know that it's particularly helpful

Reply:
lol yeah. not a big fan too

Reply:
I don't hate them =) . But I can remove this section.